### PR TITLE
Golang and grpc update [skip ci]

### PR DESCRIPTION
**Pull request by rodnymolina**
Skipping CI for now -- will launch it once the submodule changes are merged/updated.

### CODE REVIEW

File: tests/scr/testContainerInit [hunk 1]
```diff
@@ -171,11 +171,11 @@ function crio_config() {
 	# sysbox's range and allow for up to 32 containers of 65536 IDs each.
 	grep "containers" /etc/subuid
 	if [ $? -eq 1 ]; then
-		echo "containers:2147483648:2097152" >> /etc/subuid
+		echo "containers:2147483648:2097152" >>/etc/subuid
```
Comment (ctalledo) [inline tests/scr/testContainerInit:174]: Why remove the space between ">>" and "/etc/subuid"?

**Review by ctalledo — COMMENTED** [review]
Are we sure we want to update both Golang and GRPC versions just days before the release (e.g., it may destabilize things)? Should we punt this to the next release?

File: tests/scr/testContainerInit (hunk 1 above)
Comment (rodnymolina) [reply tests/scr/testContainerInit:174]: That's my bash linter, will find a way to configure it to avoid this 'improvement' ...

Comment (ctalledo) [reply tests/scr/testContainerInit:174]: Got it ...

**Comment by rodnymolina** [comment]
Yes, this golang / grpc update PR is to be merged only after our imminent release, not right now.

**Review by ctalledo — APPROVED** [review]
Approved (but please configure the bash linter to add a space after the redirection operators).

**Comment by ctalledo** [comment]
> Yes, this golang / grpc update PR is to be merged only after our imminent release, not right now.

Ok sounds good ...

**Comment by rodnymolina** [comment]
CI failure due to an unrelated issue.

**Pull request merged**